nilearn/plotting/surf_plotting.py

```diff
@@ -1644,6 +1645,24 @@ def plot_surf_roi(surf_mesh,
                          'ROI you can convert them into a ROI map like this:\n'
                          'roi_map = np.zeros(n_vertices)\n'
                          'roi_map[roi_idx] = 1')
+    if (roi < 0).any():
```

Comment:
I think you can delete the comments raising the value error here and in the tests and then just add a note above this if statement with something like `# TODO: raise ValueError in release 0.13`

And then also change the warning message to mention the version